### PR TITLE
feat(helm): better CRD handling for helm v2 and v3

### PR DESCRIPTION
**Pull request by luisdavim**
Signed-off-by: Luis Davim <luis.davim@jet.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,10 +1,9 @@
 module openebs.io/metac
 
-go 1.13.5
+go 1.13
```
Comment (luisdavim) [inline go.mod]: This denotes the minimum go language supported version and fails with go `1.14` if the minor version is included:
```
go: errors parsing go.mod:
/workingCopies/src/github.com/AmitKumarDas/metac/go.mod:3: usage: go 1.23
make: *** [Makefile:67: vendor] Error 1
```

Comment (AmitKumarDas) [reply go.mod]: Great. Thanks for the info. This is a new info to me.
We might want to put up a comment there.

**Comment by AmitKumarDas** [comment]
@luisdavim Can these get updated via a make target?
I guess I should have checked-in these in my previous PR.

In addition, is it good to have a STS of metac binary. I remember I tried to demo metac to few colleagues of mine & I was wondering if this repo had the STS spec. I would like to do this with & without helm.

File: go.mod (hunk 1 above)
Comment (luisdavim) [reply go.mod]: sure, I've added a comment.

**Comment by luisdavim** [comment]
@AmitKumarDas the `makefile` does have a target named `manifests` that will generate the CRD manifests. I've added that a while back but in this PR I'm adding that to the default target so it gets executed when you run `make` without any arguments.

**Pull request merged**